packages/web3-bzz/package.json

```diff
@@ -11,9 +11,12 @@
     "scripts": {
         "dtslint": "dtslint types --onlyTestTsNext"
     },
-    "main": "src/index.js",
+    "main": "dist/web3-bzz.cjs.js",
+    "module": "dist/web3-bzz.esm.js",
+    "unpkg": "dist/web3-bzz.min.js",
```

Comment:
Can we put some notes in the PR description above for further reference about:  
+ what each of these does 
+ how they are expected to behave
+ in which context

It can be helpful with big changes like this to have the intention 'spelled out' in the PR. That way engineers coming to this later will know exactly what was meant....

Reply:
Yes, I will add a bigger description of all the changes when I assign the PR to you for a review.